src/services/mentees.js

```diff
@@ -781,7 +782,146 @@ module.exports = class MenteesHelper {
 	}
 
 	/**
-	 * List mentees and search with name , email
+	 * Get entities and organization filter
+	 * @method
+	 * @name getFilterList
+	 * @param {String} tokenInformation - token information
+	 * @param {Boolean} queryParams - queryParams
+	 * @returns {JSON} - Filter list.
+	 */
+	static async getFilterList(queryParams, tokenInformation) {
+		try {
+			let result = {
+				organizations: [],
+				entity_types: {},
+			}
+
+			let organization_ids = []
+			const organizations = await this.getOrganizationIdBasedOnPolicy(tokenInformation.id)
+			if (organizations.success && organizations.result.length > 0) {
+				organization_ids = [...organizations.result]
+
+				if (organization_ids.length > 0) {
+					//get organization list
+					const organizations = await userRequests.listOrganization(organization_ids)
```

Comment:
Suggest that you modify the name here since it is shadowing the same variable name from parent scope.